### ISSUE TITLE
Issue with boto3 and collectfast > 1.0.0

### ISSUE THREAD

**Issue by jlariza** [antonagestam/collectfast#157]
Good day,

We have a Django project using s3 as its media/static storage engine through django storages and boto3. When using `Collectfast==1.0.0` everything works as expected. However, if we try to update the package to a new version, we get the following error when running `python manage.py collectstatic`

```
  File "/lib/python3.6/site-packages/storages/backends/s3boto.py", line 25, in <module>
    from boto import __version__ as boto_version
ModuleNotFoundError: No module named 'boto'
```

It seems that the `collectstatic` command is ignoring the storage configuration `storages.backends.s3boto3.S3Boto3Storage` and using the boto's default one.

Thank you

**Comment by antonagestam** [comment]
This should now be fixed in 1.3.0. Thanks for reporting this!

**Comment by antonagestam** [comment]
To add a bit of information here. This was a bug in the code that tries to guess which strategy to use based on the configured storage backend. Setting a value for `COLLECTFAST_STRATEGY` is a workaround and is strongly recommended anyway. I plan to remove the guessing feature in the next major release and thus require explicitly setting it.
